### DISCUSSION TITLE
Can't login after first reboot

### DISCUSSION THREAD

**Discussion by MrDerekib** [mylinuxforwork/dotfiles#322] [General]
Hi I've installed M4LW after a minimal install of arch linux (archinstall script).

I've run the package trough the paru as suggested on the Yt channel (paru -S ml4w-hyprland)

Alll ok, I can login and go to the desktop environment but when I restarted can't login anymore... go to black screen for a seconds and go back to the login screen.

Can I fix it? Maybe I have to do some modifications before restart?

It's a humble pc but it runs fine on https://github.com/prasanthrangan/hyprdots for example...

i73770k
16gb RAM
RTX3070OC
GA-Z77N Wifi MOBO

Edit: Typos

**Comment by mylinuxforwork** [comment]
Hi. Thanks for testing.

Can you try to switch to tty and try to start Hyprland with the command Hyprland.
Are there any error messages?

Do you have a NVIDIA card?

**Reply by MrDerekib** [reply]
Hey Stephan, thanks for take time to respond. I've included the "source = ~/.config/hypr/conf/environments/nvidia.conf" to the Enviroment section in hyprland.conf and now I can login and run hyprland.

What  driver are runing with this configuration? I have the feeling that some applications do not take advantage of graphics acceleration.

`> lspci -k | grep -A 3 -E "(VGA | 3D)"
01:00.0 VGA compatible controller: NVIDIA Corporation GA104 [GeForce RTX 3070] (rev a1)
	Subsystem: Gigabyte Technology Co., Ltd Device 404c
	Kernel driver in use: nouveau
	Kernel modules: nouveau`

I think the nouveau is the open source one right?

how can I add the cpu to the mission center (only cpu graph right now), taskbar info / sidebar, Alacitty (when open a new window info)?

**Reply by mylinuxforwork** [reply]
There are no drivers installed. It's just the NVIDIA environment variables.  But great that you have shared your feedback. Will invest more time into the NVIDIA setup with 2.9.7.

**Reply by MrDerekib** [reply]
I installed the nvidia-dkms and nvidia-utils drivers and when I rebooted only one of the two monitors was working and the other at low resolution and flickering. Sure I did it wrong.
Fortunately I have entered by tty and when uninstalling them it is functional again. I need a test environment and learn a lot about arch and Hyprland I think. 

A better support of Nvidia cards on 2.9.7 will be awesome!
